sidebars.js

```diff
@@ -172,6 +172,7 @@ const sidebars = {
           },
         ],
       },
+      "developers/react-development-guidelines",
```

Comment:
I don't like the current placing of this page, since I don't consider this page a first level entry in the Developers section:
![Screenshot 2023-12-05 at 14 23 51](https://github.com/multiversx/mx-docs/assets/51945539/597445ee-5499-4ca0-a4eb-d3477154aa93)

Instead, I would create a sub-category called `Guidelines` that will refer to this page, as shown in this example: 
![Screenshot 2023-12-05 at 14 26 29](https://github.com/multiversx/mx-docs/assets/51945539/57a0ec58-b77e-4178-b0b8-3b18d1960058)


In order to do so, replace:
```
"developers/react-development-guidelines",
```

with

```
{
        type: "category",
        label: "Guidelines",
        items: [
          "developers/react-development-guidelines"
        ]
      }
```